dev_scripts/convert_params.py

```diff
@@ -0,0 +1,94 @@
+"""Script converting legacy param and json parameter files to hdf5.
+This script downloads the legacy files directly from the GitHub repositories.
+"""
+# Author: George Dang <george_dang@brown.edu.com>
+
+
+import shutil
+import requests
+import os
+from pathlib import Path
+from hnn_core import convert_to_hdf5
+import tempfile
```

Comment:
Imports need to be ordered from general to more specific

system libraries -> numpy,scipy,matplotlib etc -> hnn_core